src/client/html/create.html

```diff
@@ -0,0 +1,103 @@
+                기술을 선택해주세요
+                <i class="fas fa-sort-down"></i>
+            </button>
+            <ul class="selection-container language"></ul>
+            <!-- 선택한 태그 -->
+            <ul class="tags stack-type">
+            </ul> 
+            <!-- 상세 정보 -->
+                    <!-- 기간 -->
+               <ul class="info-wrap one"> 
+                    <li class="date start">
+                        <span class="start-date">시작일</span>
+
+                        <!-- 시작일 선택 date picker -->
+                        <div class="date-picker">
+                            <div class="selected-date">2021 / 12 / 15</div>
+                            <div class="dates">
+                                <div class="month"> 
+                                    <div class="arrows prev-month">&lt;</div>
+                                    <div class="mth"></div>
+                                    <div class="arrows next-month">&gt;</div>
+                                </div>
+                                <div class="days"></div>
+                            </div>
+                        </div>
+                    </li>
+                    <dd class="date end">
+                        <span class="end-date">종료일</span>
+                        <!-- 종료일 선택 date picker -->
+                        <div class="date-picker">
+                            <div class="selected-date"></div>
+                            <div class="dates">
+                                <div class="month"> 
+                                    <div class="arrows prev-month">&lt;</div>
+                                    <div class="mth"></div>
+                                    <div class="arrows next-month">&gt;</div>
+                                </div>
+                                <div class="days"></div>
+                            </div>
+                        </div>
+                    </dd> 
+               </ul> 
+                    <!-- 지역 -->
+               <ul class="info-wrap two">
+                    <li class = "location">지역</li>
+                    <li class = "location-content">
+                        <button class="select-box location">
+                            지역 선택
+                            <i class="fas fa-sort-down"></i>
+                        </button>
+                        <!-- 셀렉트박스 요소  -->
+                        <ul class="selection-location"></ul>
+                    </li>
+                    <li class = "recruit">모집인원</li>
+                    <li class = "input-recruit">
+                        <input type="number" class="select-box number" placeholder="0" min="1" max="30" required></input>
+                    </li> 
+                </ul>
+            <textarea class = "input-text noresize" placeholder="상세 내용을 입력해주세요."></textarea>
+            <button class = "buttons submit"type="button">등록</button>
+            <button class = "buttons cancel"type="submit">취소</button>
```

Comment:
저희 컨벤션 맞춰서 클래스 띄어쓰기도 통일해주시면 좋을것 같습니다.
모든 클래스 들이 띄어쓰기가 제각각 이라 잘 통일성있게 맞춰주시면 좋을 것 같아요!
어떤 것은 띄어있고 어떤 것은 붙어져있네요
```suggestion
            <button class="buttons submit" type="button">등록</button>
            <button class="buttons cancel" type="submit">취소</button>
```